omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -28,11 +28,27 @@
             delete "#{install_dir}/bin/agent/agent.exe"
             # TODO why does this get generated at all
             delete "#{install_dir}/bin/agent/agent.exe~"
+            move "#{install_dir}/bin/agent/dist/conf.d/*.yaml*", "#{conf_dir}/conf.d/"
+             # The conf.d is already containing integration configs at this point so we have to add directories/files for core checks
+             Dir.glob("#{install_dir}/bin/agent/dist/conf.d/**/*.d").each do |check_dir|
+                dir_name = File.basename check_dir
+                mkdir "#{conf_dir_root}/conf.d/#{dir_name}" unless File.exists? "#{conf_dir}/conf.d/#{dir_name}"
+                move "#{install_dir}/bin/agent/dist/conf.d/#{dir_name}/*.yaml*", "#{conf_dir}/conf.d/#{dir_name}/", :force=>true
+            end
+            move "#{install_dir}/agent/checks.d", "#{install_dir}/checks.d"
```

Comment:
(related to my other comment: we should move this to the `datadog-agent-integrations` software def

Reply:
I put it here b/c it was done that way for linux, which will keep the `datadog-agent-integrations` def consistent across platforms.  I believe that's why @masci added this def; to keep the other base definitions more consistent and move the packaging-related layout changes all in one place, here.

Reply:
In this case I believe we should put the checks in `#{install_dir}/checks.d` directly in the `datadog-agent-integrations` software def, since the checks should be installed there whatever the platform (Linux, Windows and macOS)

Reply:
we can do this in a separate PR/later, just approved the PR!